src/adcm_pytest_plugin/plugin.py

```diff
@@ -160,4 +161,14 @@ def pytest_runtest_makereport(item, call):
     # set a report attribute for each phase of a call, which can
     # be "setup", "call", "teardown"
 
+    os.environ["rep_" + rep.when + "_passed"] = str(rep.passed)
```

Comment:
Variable name will be customized by executed procces?

Reply:
By stage. Passible stages: setup, call, teardown

Reply:
So If I run it with xdist?

Reply:
xdist starts individual processes for each worker. So they do not share env